packages/replay/src/types/performance.ts

```diff
@@ -124,6 +126,8 @@ export interface NetworkRequestData {
   statusCode?: number;
   requestBodySize?: number;
   responseBodySize?: number;
+  request?: ReplayNetworkRequestOrResponse;
+  response?: ReplayNetworkRequestOrResponse;
```

Comment:
This shouldn't belong here, this only exists on `ReplayNetworkRequestData`. `NetworkRequestData` is a deprecated interface that exists on older SDK versions.

Reply:
Update: this is correct! `ReplayNetworkRequestData` gets transformed to` ReplayPerformanceEntry<NetworkRequestData>` in `makeNetworkReplayBreadcrumb()`